src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js

```diff
@@ -236,7 +236,7 @@ define([
 
             this.sandbox.emit('husky.datagrid.' + this.options.instanceName + '.selected.update', ids);
             this.sandbox.emit('husky.datagrid.' + this.options.instanceName + '-selected.url.update', {
-                ids: ids.join(',') || ' '
+                ids: ids.join(',') || ' ' // else all are selected
```

Comment:
as shown in the issue: https://github.com/sulu/sulu/issues/2654 when you open the media selection overlay and no media is selected all are shown as selected.